src/shared/parse-helpers.c

```diff
@@ -196,3 +198,33 @@ int parse_socket_bind_item(
         *port_min = mn;
         return 0;
 }
+
+int parse_open_file_fields(const char *v, char **path, char **fdname, int *flags) {
```

Comment:
If you move `OpenFile` to parse-helpers.h, this function can be changed to `parse_open_file()` that takes a return argument of `OpenFile` instead of separate arguments for all the fields.